packages/medusa-js/src/resources/admin/auth.ts

```diff
@@ -0,0 +1,36 @@
+import { AdminAuthRes, AdminPostAuthReq } from "@medusajs/medusa"
+import { ResponsePromise } from "../../typings"
+import BaseResource from "../base"
+
+class AdminAuthResource extends BaseResource {
+  /**
+   * @description Retrieves an authenticated session
+   * Usually used to check if authenticated session is alive.
+   * @return {ResponsePromise<AdminAuthRes>}
+   */
+  getSession(): ResponsePromise<AdminAuthRes> {
```

Comment:
Would it make sense to rename to `retrieve`, `delete`, `create` for consistency; can see why this could be treated differently, so would be happy to go with it as-is if you think that's better

Reply:
In an authentication context, I would argue that the current naming is appropriate. Is also consistent with how we chose to name auth methods in the Store API.